Sources/PrivoSDK/api/Rest.swift

```diff
@@ -8,19 +8,43 @@
 import Alamofire
 import Foundation
 
+
+extension URL {
+    var status: URL {
+        return self.appendingPathComponent("status")
+    }
+    
+    var analytic: URL {
+        return self.appendingPathComponent("metrics")
+    }
+    
+    var isAnalytic: Bool {
+        return self.absoluteString.hasSuffix("metrics")
+    }
+}
+
+
 class Rest {
     
     //MARK: - Static Shared object
     
     static var shared: Rest = .init()
     
+    init(urlConfig: URLSessionConfiguration = URLSessionConfiguration.af.default) {
+        self.urlConfig = urlConfig
+        self.session = Session(configuration: urlConfig)
+    }
+    
     //MARK: - Private properties
     
     private static let storageComponent = "storage"
     private static let putComponent = "put"
     private static let sessionID = "session_id"
     private static let emptyResponsesCodes: Set<Int> = .init([200,204,205])
     
+    private let urlConfig: URLSessionConfiguration
+    private let session: Session
```

Comment:
we decided to get rid of Alamofire. Why?

Reply:
this is alamofire's session, we still use it